testnet/src/event_dispatcher.rs

```diff
@@ -86,11 +85,38 @@ impl EventObserver {
         }).to_string();
 
         // Send payload
-        let res = stream.write_bufs(&vec![payload.as_bytes().into()]);
-        if let Err(err) = res {
-            error!("Event dispatcher failed sending buffer: {:?}", err);
-            panic!();
-        }
+        let mut retry_count = 0;
+        let mut backoff: f64 = 1.0;
+        let mut rng = thread_rng();
+        loop {
+            let response = Client::new()
+                .post(&self.endpoint)
+                .json(&payload)
+                .send();
+
+            match response {
+                Ok(response) => {
+                    if response.status().is_success() {
+                        break;
+                    }
+                    error!("Event dispatcher: POST {:?} failed with error {:?}", self.endpoint, response);
+                },
+                Err(e) => {
+                    error!("Event dispatcher: POST {:?} failed with error {:?}", self.endpoint, e);
+                }
+            };
+
+            if retry_count >= self.retry_count {
+                error!("Event dispatcher: exponential backoff failed");
+                panic!();                   
+            }
+
+            retry_count += 1;
+            backoff = (2.0 * backoff + (backoff * rng.gen_range(0.0, 1.0))).min(60.0);
+            let duration = Duration::from_millis((backoff * 1_000.0) as u64);
+            debug!("Event dispatcher will retry posting in {:?}", duration);
```

Comment:
This should be a higher log level

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1535/commits/d949e080ec04d5ef2e95c2760ce4aa1f1fe7767f